service_mocks/src/main/java/uk/gov/ons/fwmt/tm_mock/controller/FieldPeriodController.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.ons.fwmt.tm_mock.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.ons.fwmt.tm_mock.dto.FieldPeriodDTO;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequestMapping("/fieldPeriods")
```

Comment:
can you add the mapping for resources service like /resources/fieldperiod

similarly for all endpoints